### ISSUE TITLE
Issue with main Phlex page?

### ISSUE THREAD

**Issue by drego05** [d8ahazard/Phlex#74]
I've posted on the forums, but i wanted to check here as well since I'm going to guess this may have come up a time or two, but i'm unable to see anything posted on it. 

I'm getting a black web page after cloning to my web servers /var/www/html directory. 

If i hit 192.168.X.X:80/Phlex  --- that's where I'm getting the black page. Thoughts?

**Comment by d8ahazard** [comment]
Try doing a pull of the latest code, I think you may have hit an issue I've since fixed.

**Comment by drego05** [comment]
Hey! Thanks for the reply!

Is it on Github? That's the one I cloned it from last week, unless you have
a direct URL to the right one.

Cheers,



Danny Rego
<http://www.linkedin.com/profile/edit?trk=hb_tab_pro_top>

**Comment by d8ahazard** [comment]
Yes, it is.  The code is updated frequently.

**Comment by drego05** [comment]
Hey d8ahazard,

As strange as it sounds, I'm still getting the same black screen. Wondering
if it's something I'm doing? I've removed all previous files of Phlex ---
cloned the directly into the /var/www/html directory --- gave the whole
directory read/write access- attempted to access the site at
mydomain.com/Phlex and it returns the black screen. I can see the title
actually indicates Phlex Web but nothing on screen. I've tried different
browsers, incognito to ensure that cookies/cache wasn't an issue... still
the same page.

Right now, I'm in the processing of updating the web server itself, but I
know the server itself is functional as I can hit other files in the root
directory with no issue ---accessing /Phlex does appear to be a problem for
some reason.

Cheers,



Danny Rego
<http://www.linkedin.com/profile/edit?trk=hb_tab_pro_top>
